scanrepository/scanrepository.go

```diff
@@ -186,24 +186,24 @@ func (cfp *ScanRepositoryCmd) getVulnerabilitiesMap(scanResults *xrayutils.Resul
 	return vulnerabilitiesMap, nil
 }
 
-func (cfp *ScanRepositoryCmd) fixVulnerablePackages(vulnerabilitiesByWdMap map[string]map[string]*utils.VulnerabilityDetails) (err error) {
+func (cfp *ScanRepositoryCmd) fixVulnerablePackages(repository *utils.Repository, vulnerabilitiesByWdMap map[string]map[string]*utils.VulnerabilityDetails) (err error) {
```

Comment:
@attiasas - please explain the change - why do we add the repo now? How did it work beforehand?
(And how is it related to the length fix?)

Reply:
@hadarshjfrog,
Now that the description for the PR we open may be limited in size, there could be a situation when we need to split the content. In order to create/update/delete comments, we need the repository object.